WhaleSpotting/Controllers/UserController.cs

```diff
@@ -26,4 +26,37 @@ public IActionResult UserPage([FromRoute] int id)
 
         return View(user);
     }
+
+    [HttpPost("{id}")]
+
+    public IActionResult EditUser
+    (
+        [FromRoute] int id,
+        [FromForm] WsUser modifiedUser
+    )
+    
+    {
```

Comment:
Bring these two onto the same line, like this:
```suggestion
    ) {
```